src/presenters/slack/enhanced-data-presenter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Yoast\WP\SEO\Presenters\Slack;
+
+use Yoast\WP\SEO\Presenters\Abstract_Indexable_Presenter;
+
+/**
+ * Presenter class for the Slack enhanced data.
+ */
+class Enhanced_Data_Presenter extends Abstract_Indexable_Presenter {
+
+	/**
+	 * Presents the enhanced data for Slack
+	 *
+	 * @return string The Twitter tags for Slack.
+	 */
+	public function present() {
+		$enhanced_data = $this->get();
+		$twitter_tags  = '';
+		$i             = 1;
+		foreach ( $enhanced_data as $label => $value ) {
+			$twitter_tags .= sprintf( "\t" . '<meta name="twitter:label%1$d" value="%2$s">' . "\n", $i, $label );
+			$twitter_tags .= sprintf( "\t" . '<meta name="twitter:data%1$d" value="%2$s">' . "\n", $i, $value );
+			$i++;
+		}
+		return \trim( $twitter_tags );
+	}
+
+	/**
+	 * Returns the enhanced data array.
+	 *
+	 * @return array
+	 */
+	public function get() {
+		$data         = [];
+		$post_content = $this->presentation->source->post_content;
+		$author_id    = $this->presentation->source->post_author;
+
+		if ( $author_id ) {
+			$data[ __( 'Written by', 'wordpress-seo' ) ] = \get_the_author_meta( 'display_name', $author_id );
+		}
+		if ( ! empty( $post_content ) ) {
+			$data[ __( 'Est. reading time', 'wordpress-seo' ) ] = $this->get_reading_time( $post_content );
+		}
+		return \apply_filters( 'wpseo_enhanced_data', $data, $this->presentation );
+	}
+
+	/**
+	 * Calculate the estimated reading time.
+	 *
+	 * @param string $post_content The post content.
+	 * @return string Human-readable est. reading time.
+	 */
+	private function get_reading_time( $post_content ) {
```

Comment:
Is there a specific reason for this method to be private? If not, please change this and add tests.

Reply:
I made the opposite reasoning: there is no reason to make it public. In all the presenters, methods (except `present()` and `get()`) are private or protected, see e.g.:
https://github.com/Yoast/wordpress-seo/blob/85c4c275f4dce333c2ef5c1e6dc47517cb07797e/src/presenters/googlebot-presenter.php#L56
https://github.com/Yoast/wordpress-seo/blob/85c4c275f4dce333c2ef5c1e6dc47517cb07797e/src/presenters/twitter/site-presenter.php#L53